test/entrypoint.test.ts

```diff
@@ -1009,7 +1008,7 @@ describe('EntryPoint', function () {
             paymasterAndData: hexConcat([paymaster.address, expireTime])
           }, ethersSigner, entryPoint)
           const ret = await entryPoint.callStatic.simulateValidation(userOp).catch(simulationResultCatch)
-          expect(ret.returnInfo.paymasterDeadline).to.eql(now - 60)
+          expect(ret.returnInfo.validUntil).to.eql(now - 60 - 1)
```

Comment:
We have tests for expiration `validUntil` for both Account and Paymaster, but no tests for `validAfter` for either, as far as I can tell.  Or a test that includes both for that matter (an op that becomes valid in a future block, and invalid in a later block).

Reply:
yes, I need to add an explicit test for validAfter.